pkg/webhooks/clusterqueue_webhook.go

```diff
@@ -127,7 +127,7 @@ func validatePreemption(preemption *kueue.ClusterQueuePreemption, path *field.Pa
 
 func validateCQAdmissionChecks(spec *kueue.ClusterQueueSpec, path *field.Path) field.ErrorList {
 	var allErrs field.ErrorList
-	if len(spec.AdmissionChecksStrategy.AdmissionChecks) != 0 && len(spec.AdmissionChecks) != 0 {
+	if spec.AdmissionChecksStrategy != nil && len(spec.AdmissionChecks) != 0 {
```

Comment:
Could you update this unit test like `AdmissionCheckStrategy()`?

https://github.com/kubernetes-sigs/kueue/blob/3112eec330ee88f153558c68636380658bcf917c/pkg/webhooks/clusterqueue_webhook_test.go#L76-L77

Reply:
Could you clarify please? Do you mean to not include any rules in the `AdmissionCheckStrategy` field?

Reply:
Yes, that's right. I would like to prove if the webhook error happens regardless of the rules.